### ISSUE TITLE
Issue when rendering other players causing freezes and crashes

### ISSUE THREAD

**Issue by Smig123** [Direwolf20-MC/BuildingGadgets#343]
TL:DR - We're having an issue where players will freeze for 4 seconds when rendering each other. The problem is beginning to cause crashes. 

So a few weeks ago me and a bunch of friends started a highly edited version of the FTB Direwolf20 Pack. What we did was delete most of the mods in the pack and then added a bunch of our own and then hosted our own server for the modpack for just us to play on. Since day one however, we have had an issue which causes players to freeze for 4 seconds when they render each other in. Every time a players leaves the chunks in your render distance, whenever they return you and the person returning will experience the freeze. We're mostly split up in groups too and whenever we warped between each others bases everyone at the base would freeze due to them having to render in the player warping. While this was very frustrating, we eventually got a bit used to it and some of us just increased our render distance to keep each other rendered all the time. However the problem seems to have gotten worse and its now causing permanent freezes until one person crashes. I was wondering if anyone else has had this problem before and if so are there any solutions? So far we have tried installing Optifine and Texfix RAM Optimisation mod to see if it would help. Any suggestions would be appreciated very much :)
